vdp/pipeline/v1beta/pipeline.proto

```diff
@@ -1984,6 +1984,26 @@ message ListPipelineRunsRequest {
   optional string order_by = 7 [(google.api.field_behavior) = OPTIONAL];
 }
 
+// ListPipelineRunsByCreditOwnerRequest is the request message for ListPipelineRunsByCreditOwner.
+message ListPipelineRunsByCreditOwnerRequest {
```

Comment:
The API design is a bit unclear to me if the endpoint is called `ByCreditOwner` but we don't have a request field to specify the owner. I see it's documented as the requester header, but
- That means it'll only be documented for HTTP requests
- It forces the user to know the requester UID. Public endpoints should expose the public ID instead, if possible.

Is there a reason to use the header param instead of a `requester_id` field?

Reply:
Will be renamed to `ListPipelineRunsByRequester`.
If users did not provide a requester UID, in backend the user UID extracted from token will be used.
If users provide a org/namespace UID, the org/namespace UID will be used

Reply:
If we introduce a `requester_id` parameter, I would make it required and rely only on that to extract the requester ID. If we have a clear way to do it, I wouldn't add layers of complexity. The user UID in the token and in the requester header should only be used to check permissions.

Reply:
This data should only be accessible to the individual user. If we allow API users to modify the value via a `requester_id` parameter, is it possible that an API user could retrieve someone else's user ID or UUID and view their credit usage records?